### PR TITLE
Remove all available 9999 now that 10.15, iOS 13, etc are public.

### PR DESCRIPTION
**Pull request by stephentyrone**

### CODE REVIEW

**Comment by stephentyrone** [comment]
@jckarter please confirm that the availability updates in test/Interpreter and test/ParseableInterfaces are correct.

**Comment by stephentyrone** [comment]
@swift-ci please test

**Review by jckarter — APPROVED** [review]
Thanks Steve! Looks good.

**Review by lorentey — APPROVED** [review]
LGTM! 

The patch adds some extra newlines in unrelated files. I think we should keep them.

**Comment by lorentey** [comment]
Master is now building with Xcode 11; this will have a better chance of passing tests now.

**Comment by lorentey** [comment]
@swift-ci test macOS platform

**Comment by stephentyrone** [comment]
@swift-ci test macOS

**Comment by stephentyrone** [comment]
@swift-ci please test

**Comment by stephentyrone** [comment]
@swift-ci smoke test macOS

**Comment by stephentyrone** [comment]
@swift-ci smoke test macOS

**Comment by stephentyrone** [comment]
@swift-ci please smoke test

File: stdlib/public/core/CollectionDifference.swift [hunk 1]
```diff
@@ -222,7 +222,11 @@ public struct CollectionDifference<ChangeElement> {
 ///   }
 /// }
 /// ```
+<<<<<<< HEAD
```
Comment (yas375) [inline stdlib/public/core/CollectionDifference.swift]: Probably wasn't intended to be committed. There are a few of them.

Comment (stephentyrone) [reply stdlib/public/core/CollectionDifference.swift]: Yeah, I’m reworking some of this anyway to minimize the churn.

**Comment by stephentyrone** [comment]
Closing this in favor of simpler https://github.com/apple/swift/pull/26108 + follow-on PR for `test/*`.

**Pull request closed without merging**